includes/Modules/Site_Verification.php

```diff
@@ -75,10 +75,13 @@ function() {
 		add_action(
 			'init',
 			function () {
+				$request_uri    = $this->context->input()->filter( INPUT_SERVER, 'REQUEST_URI' );
+				$request_method = $this->context->input()->filter( INPUT_SERVER, 'REQUEST_METHOD' );
```

Comment:
There are some known issues regarding `INPUT_SERVER` and `INPUT_ENV` in certain setups, I ran into this in another project. See e.g. https://stackoverflow.com/questions/25232975/php-filter-inputinput-server-request-method-returns-null for some information

We could either keep the `$_SERVER` accesses untouched for now, or add internal logic to the `Input` class to work around this issue. What do you think?

Reply:
Good catch! I wasn't aware of this. I would prefer to use the same utility in all cases. This wouldn't impact `MutableInput` so we only need to patch `Input` to ensure this works. I'll see what it would take to add 👍 

I think worst case, we could attempt to mimic the immutable behavior of `filter_input` by taking a copy of those particular superglobals when the class is instantiated. Then we could use `filter_var` internally like we do with `MutableInput`. Ideally we could detect when this would be necessary, if not, we can either try `filter_input` first with a fallback or just use the alternate method; with these input types only of course.